TodoList.java

```diff
@@ -23,19 +24,52 @@ public void removeTasks(String task) {
     }
   }
 
-  // view Task ---
-  public void ViewTasks(String task) {
+  // Imporved View All Tasks
+
+  public void viewAllTasks() {
     if (tasks.isEmpty()) {
-      System.out.println("No tasks in the list.");
+      System.out.println("No tasks in the list");
     } else {
+      System.out.println("----- All Tasks -----");
       for (int i = 0; i < tasks.size(); i++) {
         System.out.println((i + 1) + "." + tasks.get(i));
       }
     }
   }
 
-  // dummy print
-  public void print() {
-    System.out.println("Greetings!!");
+  // View Completed Tasks
+  public void viewCompletedTasks() {
+    ArrayList<String> completedTasks = new ArrayList<>();
```

Comment:
Move the complete task code to another method. Example : getCompletedTask(List<String> tasks){
for (String task : tasks) {
      if (task.endsWith("[Completed]")) {
        completedTasks.add(task);
      }
    }
}